src/lib/rest/rest.cpp

```diff
@@ -91,7 +91,7 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
   std::string      key   = ckey;
   std::string      value = (val == NULL)? "" : val;
 
-  if (*val == 0)
+  if (val == NULL || *val == 0)
   {
     OrionError error(SccBadRequest, std::string("Empty right-hand-side for URI param /") + ckey + "/");
```

Comment:
Not sure this the error that we want to respond with in the case of 'no right hand side nor equal-sign'.
Would this be more of a "URI parameter /xxx/ not supported"?

E.g.

    if (val == NULL)
    {
      OrionError error(SccBadRequest, std::string("invalid URI param /") + ckey + "/");
￼      ...
    }

Open for discussion ...